tests/ui/manual_split_once.stderr

```diff
@@ -22,13 +22,13 @@ error: manual implementation of `split_once`
   --> $DIR/manual_split_once.rs:17:13
    |
 LL |     let _ = "key=value".splitn(2, '=').nth(1).unwrap();
-   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `"key=value".split_once('=').unwrap().1`
+   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `"key=value".split_once('=').map_or("key=value", |x| x.1)`
```

Comment:
This changes behavior since `nth(1)` is not guaranteed to return `Some`. The issue in question is `[r]splitn(..).next().unwrap()` which never panics because `splitn` always has at least one element.